Author: Pino Toscano <pino@debian.org>
Description: Disable tests that require networking
 Networking is not available during package building on buildds, so skip the
 checks that require network access.
Last-Updated: 2018-05-26
Forwarded: not-needed

Index: b/src/tests/tellicoreadtest.cpp
===================================================================
--- a/src/tests/tellicoreadtest.cpp
+++ b/src/tests/tellicoreadtest.cpp
@@ -258,6 +258,7 @@ void TellicoReadTest::testLocalImage() {
 }
 
 void TellicoReadTest::testRemoteImage() {
+#if 0
   // this is the md5 hash of the logo.png icon, used as an image id
   const QString imageId(QL1("757322046f4aa54290a3d92b05b71ca1.png"));
   // not yet loaded
@@ -290,6 +291,9 @@ void TellicoReadTest::testRemoteImage()
 
   const Tellico::Data::Image& img = Tellico::ImageFactory::imageById(imageId);
   QVERIFY(!img.isNull());
+#else
+  QSKIP("disabled network test");
+#endif
 }
 
 void TellicoReadTest::testXMLHandler() {
Index: b/src/tests/imagejobtest.cpp
===================================================================
--- a/src/tests/imagejobtest.cpp
+++ b/src/tests/imagejobtest.cpp
@@ -191,6 +191,7 @@ void ImageJobTest::testImageLink() {
   QCOMPARE(img.linkOnly(), true);
 }
 
+#if 0
 void ImageJobTest::testNetworkImage() {
   if(!networkIsAvailable()) {
     QSKIP("This test requires network access", SkipSingle);
@@ -371,3 +372,4 @@ void ImageJobTest::testFactoryRequestNet
   QCOMPARE(img.format(), QByteArray("png"));
   QCOMPARE(img.linkOnly(), true);
 }
+#endif
Index: b/src/tests/imagejobtest.h
===================================================================
--- a/src/tests/imagejobtest.h
+++ b/src/tests/imagejobtest.h
@@ -44,6 +44,7 @@ private Q_SLOTS:
   void testImageLoad();
   void testImageLoadWithId();
   void testImageLink();
+#if 0
   void testNetworkImage();
   void testNetworkImageLink();
   void testNetworkImageInvalid();
@@ -51,6 +52,7 @@ private Q_SLOTS:
   void testFactoryRequestLocalInvalid();
   void testFactoryRequestNetwork();
   void testFactoryRequestNetworkLinkOnly();
+#endif
 
 Q_SIGNALS:
   void exitLoop();
